bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandle.java

```diff
@@ -1972,6 +1972,11 @@ void handleBookieFailure(final Map<Integer, BookieSocketAddress> failedBookies)
         }
     }
 
+    boolean testStubResolveConflict() {
+        // No test inserts by default
+        return false;
+    }
+
```

Comment:
I did not like the stub function to inject mock object, but I ran into various complications with mocking inner class methods which are generated on demand. If there is any better way I am more than happy to adopt it. Given that this is not applicable for master I gave pull request to 4.8 only. Not sure if we have a test case in the master to cover this case. @ivankelly ?